frontend/src/components/CreateNewSongbook.tsx

```diff
@@ -0,0 +1,87 @@
+import { Box, Button, Checkbox, Heading, Input, Text } from "@chakra-ui/react";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { createNewSongbook } from "../services/songs";
+
+export default function CreateNewSongbook() {
+  const [key, setKey] = useState("");
+  const [maxSongs, setMaxSongs] = useState("");
+  const [title, setTitle] = useState("");
+  const [noodleMode, setNoodleMode] = useState(false);
+  const navigate = useNavigate();
+  const parsedSongCap = parseInt(maxSongs);
+  return (
+    <Box padding="2rem">
+      <Heading size="lg" mb="3rem">
+        Create a New Songbook:
+      </Heading>
+      <form>
+        <Text mb="1rem">
+          TITLE (<i>displays on songbook</i>):
```

Comment:
This is kind of nitpicky but I don't prefer the all-caps titles. I think it's a bit of an old-school form look.

Reply:
Roger. Fixed.